DEVELOPING.rst

```diff
@@ -8,461 +8,91 @@
-
-Working with iSpy visualizer
-----------------------------
-
-CSS dependencies which are needed for iSpy CMS visualizer are sandboxed in order
-to make it compatible with ``Semantic UI``. This was achieved by:
-
-* Wrapping all the ``Bootstrap`` html with a ``<div class="bootstrap-ispy">``
-* Prefixing all the css classes of ``Bootstrap`` framework and custom ispy css file with ``bootstrap-ispy`` class.
-* As a result ``Bootstrap`` css can be used inside a div with ``bootstrap-ispy`` class without any conflicts with ``Semantic UI``.
-
-Procedure to prefix css files with ``bootstrap-ispy`` class:
-
-* Download unminified version (CMS visualizer currently uses Bootstrap v3.3.1) of the ``Bootstrap`` framework from the official website (usually it's bootstrap.css file)
-* Install LESS preprocessor locally: ``npm install -g less``
-* Create a file ``prefix-bootstrap.less`` which contains the following:
-
-.. code-block:: console
-
-   .bootstrap-ispy {
-      @import (less) 'bootstrap.css';
-   }
-
-* Preprocess css file with LESS to generate a new prefixed file (it will create ``bootstrap-prefixed.css`` file):
+For a quickstart guide, do the following:
 
 .. code-block:: console
 
-   lessc prefix-bootstrap.less bootstrap-prefixed.css
+  $ # Checkout this repository
+  $ git clone https://github.com/cernopendata/opendata.cern.ch.git
+  $ # Checkout the module with the portal
+  $ git clone https://github.com/cernopendata/cernopendata-portal.git
+  $ # Move to the directory of the content
+  $ cd opendata.cern.ch
+  $ # The command line is a bit long: let's create an alias to avoid typing all of that
+  $ alias my_docker="docker compose -f ../cernopendata-portal/docker-compose.yml -f docker-compose-content.yml"
+  $ # Make sure that the latest images are available
+  $ my_docker pull
+  $ # Start the services
+  $ my_docker up -d
+  $ # Give enough time to the containers to start properly. Note that there are some dependencies among them,
+  $ # and the web container starts by setting up the development environment
+  $ sleep 120
+  $ # Create the basic structure
+  $ my_docker exec -it web /code/scripts/populate-instance.sh
 
-* Place this file in ``/static/assets/`` to serve it
-* Same exact procedure needs to be done for custom `ispy.css file <https://github.com/cms-outreach/ispy-webgl/blob/master/css/ispy.css>`_
+..
 
-Switching between PROD and DEV contexts
----------------------------------------
 
-If you need to switch between testing a feature is the development environment
-context (using ``docker-compose-dev.yml``) and the production environment
-context (using ``docker-compose.yml``), you can use a helper script joining the
-above tips together to quickly initialise your working environment.
+At this point, all the services should be up and running. If you go to a web browser to http://0.0.0.0:5000/, you should
+see the web portal, with a few entries inside (the vocabularies, and news about the portal itself).
```

Comment:
What terminology do you suggest for something that includes records/documents/glossary terms? Specifying the three would be too much.

Reply:
I would keep the three sections separated, as is the case now. When someone work with docs, we need to explain how to use Markdown, use LaTeX, how to add static images, etc. These could live under the documentation sertion, because we don't support Markdown in records, etc. Just an example why these were separated sections before. (BTW you have removed those parts?)

As for the terms, these are living in the `cernopendata-portal` repository, so we should just point there how to add terms, load, check, etc. So the guide won't be detailed here.  (Although IMO the glossary terms should live with the content, not with the infrastructure.)